package.json

```diff
@@ -64,17 +69,13 @@
     "query-string": "^2.4.1",
     "randgen": "^0.1.0",
     "react": "0.13.3",
-    "should": "^7.1.0",
     "sprintf": "^0.1.5",
-    "topojson": "^1.6.19",
```

Comment:
I wish there was some way to add a comment to package.json. I have also tried once to move topojson to devDependencies.